spec/include_directive/parser_spec.rb

```diff
@@ -1,3 +1,4 @@
+require_relative '../../spec/spec_helper'
```

Comment:
Doesn't work on my system, but I noticed it's more complicated than I intended. `require_relative '../spec_helper'`